.azure/pipelines/components-e2e-tests.yml

```
@@ -44,9 +44,9 @@ jobs:
       displayName: NPM install
     - script: .dotnet/dotnet build ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-restore
       displayName: Build
-    - script: .dotnet/dotnet test ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-build --filter QuarantinedTest\!~github --logger trx
+    - script: .dotnet/dotnet test ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-build --filter Quarantined\!=true --logger trx
       displayName: Run E2E tests
-    - script: .dotnet/dotnet test ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-build --filter QuarantinedTest~github -p:RunQuarantinedTests=true --logger trx
+    - script: .dotnet/dotnet test ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-build --filter Quarantined=true -p:RunQuarantinedTests=true --logger trx
```

```suggestion
    - script: .dotnet/dotnet test ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-build --filter Quarantined=true -p:RunQuarantinedTests=true --logger trx --logger:"trx%3BLogFileName=$(_TestResultTrxFileName)" --logger:"html%3BLogFileName=$(_TestResultHtmlFileName)" "--ResultsDirectory:$(_TestResultDirectory)"`
```

Up to you whether the HTML logger is needed and how you want to set the filenames. See src/Microsoft.DotNet.Arcade.Sdk/tools/VSTest.targets and src/Microsoft.DotNet.Arcade.Sdk/tools/Tests.targets in the Arcade repo for inspiration.